Sources/HaCWebsiteLib/Controllers/LandingPageController.swift

```diff
@@ -7,37 +7,23 @@ import DotEnv
 import SwiftyJSON
 
 struct LandingPageController {
-  static let updates = [
-    PostCard(
-      title: "Conquering Linux",
-      category: .workshop,
-      description: "Be great at computers, not because you can, but because you dream.",
-      backgroundColor: "purple",
-      imageURL: "/static/images/workshop.jpg"
-    ),
+
+  static let eventPostCards : [PostCard] = EventManager.events.flatMap { event in
+    return event.toPostCard()
```

Comment:
Nit: Return is superfluous here. If you're feeling really adventurous, this entire closure can be written as `{ $0.toPostCard() }` (it varies from case to case whether that is more readable)